src/simd_fnt.h

```diff
@@ -473,9 +473,8 @@ inline void encode_post_process(
     T threshold,
     size_t vecs_nb)
 {
-    const unsigned element_size = sizeof(T);
     const unsigned vec_size = countof<T>();
-    const T max = 1 << (element_size * 8 - 1);
+    const T max = 1U << (sizeof(T) * 8 - 1);
```

Comment:
minor: could use CHAR_BIT instead of 8 (to be 100% portable, even so 8 is probably enough for the platform we target).